src/explore-education-statistics-frontend/src/modules/find-statistics/components/DataBlock.tsx

```diff
@@ -134,7 +135,7 @@ export class DataBlock extends Component<DataBlockProps, DataBlockState> {
             </TabsSection>
           )}
 
-          {this.state.tables && (
+          {this.state.tables && this.props.showTables !== false && (
```

Comment:
Nitpick but `&& this.props.showTables &&` should be sufficient (as it's a boolean).

Reply:
It might not be set, i.e. undefined, hence the !==

Reply:
Fair enough, but I would suggest opting to use a default prop value instead to be a bit clearer about the intention here (would also make it sufficient to use the truthy coercion).